xds/third_party/envoy/import.sh

```diff
@@ -20,7 +20,7 @@
 set -e
 BRANCH=master
 # import VERSION from one of the google internal CLs
-VERSION=109d23a6648b1ac7723c4b2f125e913125bb9a84
+VERSION=5a92867ab5437861b072921b11e9083cf0c63aa5
```

Comment:
Does this commit match any internal CL?

Reply:
No, this commit is newer than the one in latest internal CL

Reply:
If your code will use that field in generated code, the grpc-java import to google3 will break. Can you wait for the next envoy internal CL (and ask the envoy import oncall to do their import soon) ?

Reply:
Sure, we can wait for it.

Reply:
@dapengzhang0 I updated the PR and import with the commit that matches the latest internal envoy import. Now we can merge.